tests/jest/unit/assets/dev/js/utils/time.test.js

```diff
@@ -0,0 +1,12 @@
+// Arrange.
+import getUserTimestamp from 'elementor-utils/time';
+
+describe( 'Time Utility', () => {
```

Comment:
Please fix the code and the test according to:
https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters#:~:text=Testable%20and%20Untestable%20Code

Reply:
Done